### PR TITLE
clang-format: copy latest and re-format

### PR DESCRIPTION
**Pull request by geissonator**
clang-format-16 has some backwards incompatible changes that require additional settings for best compatibility and re-running the formatter. Copy the latest .clang-format from the docs repository and reformat the repository.

Change-Id: Ia442cdc88aaeaab2f1384cc46a56f5cd5e40f2f5

### CODE REVIEW

**Comment by geissonator** [comment]
Upstream commit: https://gerrit.openbmc.org/c/openbmc/openpower-vpd-parser/+/63255

**Comment by geissonator** [comment]
jenkins run tests please

**Review by jinuthomas — APPROVED** [review]

File: ibm_vpd_utils.cpp [hunk 1]
```diff
@@ -700,17 +701,30 @@ std::string getPrintableValue(const std::variant<Binary, std::string>& kwVal)
...
+            if (printable)
             {
-                kwString = std::string(kwVal.begin(), kwVal.end());
+                kwString = std::string(kwVal.begin(), it);
             }
+        }
+        else
+        {
+            kwString = std::string(kwVal.begin(), kwVal.end());
+        }
```
Comment (GiridhariKrishnan) [inline ibm_vpd_utils.cpp:727]: Hi Andrew,
These changes have reverted to old one, so the defect may not be fixed. Can you please change it to correct one, if possible. Thanks.

Comment (geissonator) [reply ibm_vpd_utils.cpp:727]: Not sure I understand @GiridhariKrishnan , this clang format run was based on what was in 1050 when I ran it. Did you have a change come in after that changed this code in some way? If so, a merge conflict should have come for one of us depending on who merged first (and then we'd be forced to fix it).

**Comment by jinuthomas** [comment]
@GiridhariKrishnan I thought I did the changes, could you look at the latest file in 1050 and see if it is correct or not.
@geissonator  , the change he is saying is that his commit removes some code and in upstream the commit is merged after Patricks commit of clang 16, whereas in downstream it is merged before your commit. of clang 16. I have reapplied his commit in downstream , so all should be good IMO unless Siri comes back saying I missed something.

**Pull request merged**